entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+set -e
+
+NB_POSTGRESQL_DB=pfb
+NB_POSTGRESQL_USER=gis
+NB_POSTGRESQL_PASSWORD=gis
+
+# start postgres and capture the PID
+/docker-entrypoint.sh postgres &
+POSTGRES_PROC=$!
+
+# sleep while db comes up
+sleep 10
```

Comment:
This needs to be replaced with something that tests if the database is ready. Considering pulling in https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh